sampleconfig/core.yaml

```diff
@@ -454,13 +454,11 @@ peer:
     #     library: /etc/hyperledger/fabric/plugin/escc.so
     handlers:
         authFilters:
-          -
-            name: DefaultAuth
-          -
-            name: ExpirationCheck    # This filter checks identity x509 certificate expiration
+          - name: DefaultAuth
```

Comment:
While I'm fine with implementing the filters, I don't think we should activate them by default. Let's keep them commented out in the sampleconfig.